src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2049,6 +2049,33 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         //   the \n (but not including it). Then, we check the if the
         //   previous character is \r, if its not, then we had a lone \n
         //   and so we append our own \r
+        // - For security reasons, most control characters should be removed.
+
+        auto pred = [](wchar_t c) {

```

Comment:
Inlining this in `std::remove_if` results in very weird indention using `clang-format`. Nesting this inside `sanitize` crashes `clang-format`(!). So I had to put it here.